src/modules/Dial.js

```diff
@@ -10,34 +11,77 @@
 
 const Timezone = require('moment-timezone');
 
+function getElementByClassOrId(elementName, parentId, defaultElementName) {
+  if (!elementName && !parentId)
+    return null;
+
+  const elementNameAsId = elementName ? "#" + elementName : "#" + defaultElementName;
+  const elementNameAsClass = elementName ? "." + elementName : "." + defaultElementName;
+  const parentIdSelector = "#" + parentId;
+  let element;
+
+  if (parentId) {
+    element = document.querySelector(parentIdSelector + " " + elementNameAsClass) || document.querySelector(parentIdSelector + " " + elementNameAsId);
+
+    if (!element && elementName) {
+      // Log it and try to retrieve it by id in the whole document instead as last resource
+      console.warn("Element '" + elementName + "' could not be found inside '" + parentId + "'");
+      element = document.getElementById(elementName);
+    }
+  } else {
+    element = document.getElementById(elementName);
+  }
+
+  return element;
+}
+
 class Dial {
-  constructor(settings, parentWatch) {
-    this.error = false;
-    this.errorChecking(settings);
+  constructor(settings, parentWatch, parentId) {
```

Comment:
If the parentId exists on the parent watch class, then it would be accessible from the parentWatch parameter already `parentWatch.id`;